packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -22,6 +22,7 @@ const {
 } = require(`./hast-processing`)
 const codeHandler = require(`./code-handler`)
 const { timeToRead } = require(`./utils/time-to-read`)
+const { getHeadingID } = require(`./utils/get-heading-id`)
```

Comment:
I think we are missing `utils/get-heading-id.js` file in the PR/branch now ;)

But this is what I meant in general, so thanks for doing that!

Reply:
ops, someone have `utils` in `.gitignore`. I didn't notice this.

Reply:
Ah, you are right! Sorry about that - there also seem to be some shuffling around this recently ( some PR completely removed that and then this one tries to re-add it https://github.com/gatsbyjs/gatsby/pull/23582 )